priv/repo/migrations/20241211135758_certifications.exs

```diff
@@ -0,0 +1,18 @@
+defmodule Orbit.Repo.Migrations.CertificationsTable do
+  use Ecto.Migration
+
+  def change do
+    create table(:certifications) do
+      add(:badge, :string)
+      # Enum
+      add(:type, :integer)
+      # Enum
+      add(:rail_line, :rail_line)
+      add(:expires, :date)
+      timestamps()
+    end
+
+    create unique_index(:certifications, [:badge, :type])
```

Comment:
Related to the comment in `certification.ex`, should this also have `rail_line` so employees can be certified on more than one line?

Reply:
yes 👍 good callout.... https://github.com/mbta/orbit/pull/203/files#diff-248c4b179027894763ea37f20c43c9e2ac96ea5b22619c4e898777d809a5c8b5R15